modules/space/horizonsfile.cpp

```diff
@@ -504,8 +504,9 @@ HorizonsResult readHorizonsFile(std::filesystem::path file) {
     // " Before data starts, Observer table doesn't
     std::string line;
     std::getline(fileStream, line);
+    const std::string vectorTabHeader = "JDTDB";
     while (line[0] != '$') {
-        if (line == "JDTDB") {
+        if (line.substr(0, vectorTabHeader.size()) == vectorTabHeader) {
```

Comment:
```suggestion
        if (line.starts_with(VectorTabHeader)) {
```

Shiny new C++20

or
`if (line.starts_with("JDTBD"))`

Reply:
Switched to the cool new way